frontend/packages/dev-console/integration-tests/features/topology/topology-smoke_test.feature

```diff
@@ -0,0 +1,165 @@
+   And user sees build decorator on bottom left which will take user to either build tab or pipeline depending on pipeline associated with them 
+   Then user checks node label having "D" for deployment and then name of node
+
+@regression, @smoke
+Scenario: Visual for deployment-config
+   Given user is in topology with deployment-config workload
+   When user checks nodes and the decorators associated with them
+   And nodes are circular shaped with builder image in them
+   And pod ring associated with node are present around node with color according to the pod status
+   And deployment-config can have application url on top-right of the node 
+   And user sees edit source code decorator is on bottom right of the node which can lead to github or che workspace
+   And user sees build decorator on bottom left which will take user to either build tab or pipeline depending on pipeline associated with them 
+   Then user checks node label having "DC" for deployment-config and then name of node
+
+@regression, @smoke
+Scenario: Visual for knative service with no revision
+   Given user is in topology with knative workload
+   When user checks nodes and the decorators associated with them
+   And user can view knative service are rectangular shaped with round corners
+   And user can see dotted boundary with text "No Revision" mentioned
+   And knative sevice app can have application url on top-right of the node
+   And user sees build decorator on bottom left on knative service app which will take user to build tab
+   Then user checks knative service having label "KSVC" and then the name of service
+
+@regression, @smoke
+Scenario: Visual for knative service with revisions
+   Given user is in topology with knative workload
+   When user checks nodes and the decorators associated with them
+   And user can view knative service are rectangular shaped with round corners
+   And user can see knative service app with dotted boundary with revision present inside it
+   And knative sevice app can have application url on top-right of the node
+   And user can see traffic distribution from knative sevice app to its revisions with it's percentage number
+   And pod ring associated with revisions are present around node with color according to the pod status
+   And knative revision can have application url on top-right of the node 
+   And user sees edit source code decorator is on bottom right of the revision which can lead to github or che workspace
+   And user sees build decorator on bottom left on knative service app which will take user to either build tab 
+   And user checks revisions having label "REV" and then the name
+   Then user checks knative service having label "KSVC" and then the name of service
+
+@regression, @smoke
+Scenario: Sidebar of worload
+   Given user is in the topology with workloads
+   When user clicks on workload
```

Comment:
```suggestion
   When user clicks on workload to open the sidebar for workload
```

Reply:
This scenario is to verify sidebar so it can not be written in when statement